### DISCUSSION TITLE
Key Bindings option to Toggle HDR

### DISCUSSION THREAD

**Discussion by Valour549** [seerge/g-helper#3328] [General]
The standard hotkey for HDR toggling is Win+Alt+B, but that only works if Xbox Game Bar is installed. But ofc installing Game Bar means a lot of your hotkeys combinations are hijacked with no way to disable them, and there are better programs to record games.

I haven't found another way to toggle HDR without a lot of clicking through system settings, it would be awesome if it becomes an option in the custom Key Bindings.

**Comment by seerge** [comment]
@Valour549 hello, but why don't you just use `Win+Alt+B` then (with game bar)? Cause as you correctly say - it doesn't seem to be any other "official" way to toggle it, so most likely you can't do it programmatically.

**Reply by Valour549** [reply]
The problem with installing Game Bar is that it automatically hijacks six hotkey combinations with no option to change or disable them, and it's inferior to OBS and Nvidia app with regards to game recording in the first place.

[HDR Tray](https://github.com/res2k/HDRTray) works perfectly, but it's one additional program and it doesn't exactly have a [hotkey](https://github.com/res2k/HDRTray/issues/19). I'm sure if you have a look you'll be able to integrate it into G-Helper.
